integration_tests/suite/helpers/websocketd.py

```diff
@@ -17,7 +17,7 @@ class WebSocketdClient(object):
     _DEFAULT_TIMEOUT = 5
 
     def __init__(self, loop, port):
-        self._loop = loop
+        self._loop = loop or asyncio.get_event_loop()
```

Comment:
To be more intuitive when instantiate the `WebSocketdClient(443)` with `loop=None`
```suggestion   
def __init__(self, port, loop=None):
        self._loop = loop or asyncio.get_event_loop()
```